### ISSUE TITLE
Fix aria-describedby in form help text and error messages

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#731]
## Describe the bug
`aria-describedby` is an attribute that should be used on the input element that points to an id on the help text element, not the other way around. In DIMES, this has been implemented backwards so that the help text has the `aria-describedby` attribute instead of the input.

## To reproduce
1. Go to the request in reading room or request copies modal
2. Inspect the page
3. See that the help text below text inputs and text areas has an aria-describedby attribute, but the input itself does not.
4. Try submitting a form without filling it out
5. See that the error message below the inputs have an aria-describedby attribute, but the inputs do not.

## Expected behavior
See the [RAC style library text area with help text](https://styles.rockarch.org/?path=/story/components-form-textarea-inputs--help-text) and [with error](https://styles.rockarch.org/?path=/story/components-form-textarea-inputs--with-error) for the correct implementation. 

## Impact on your work
When screen reader users navigate to a form input, they won't automatically hear the associated help text or error messages.

**Comment by HaSistrunk** [comment]
Update: this appears to currently only be an issue for the "Request in Reading Room" scheduled date input help text.
